src/engraving/dom/harppedaldiagram.h

```diff
@@ -82,6 +82,7 @@ class HarpPedalDiagram final : public TextBase
     void updateDiagramText();
 
     bool isTpcPlayable(int tpc);
+    std::set<int> getPlayableTpcs() { return m_playableTpcs; }
```

Comment:
Let's stick to our habit of not using `get` prefixes:
```suggestion
    std::set<int> playableTpcs() { return m_playableTpcs; }
```